### PR TITLE
docs(server): add impersonation docs

### PR DESCRIPTION
**Pull request by BarbecueSilver**
* how to restrict impersonation to specific users via `impersonationAuthorize`
* how to check if a networkSession is impersonated

---

Related to discussion https://github.com/accounts-js/accounts/discussions/1195

### CODE REVIEW

**Comment by pradel** [comment]
@BarbecueSilver I feel like all this part should be moved on the server page rather than the client. The client could reference that part of the server page tho to make it easier to find.

**Comment by BarbecueSilver** [comment]
@pradel good point. Will move it there :)

**Comment by BarbecueSilver** [comment]
@pradel good 2 go :rocket:

File: website/docs/server.md [hunk 1]
```diff
@@ -123,6 +144,31 @@ Logout a user and invalidate their session with the session access token.
 await accountsServer.logout(accessToken);
 ```
 
+### Check if session is impersonated
```
Comment (pradel) [inline website/docs/server.md:147]: What do you think about moving the "Authorize to impersonate" paragraph at the same level as here?

File: website/docs/client.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ await accountsClient.getUser();
 
 ### Impersonating another user
 
-If you need to you can allow some users to impersonate the other users accounts eg: only admin users can impersonate to other users.
-Your current session will be replaced by another session.
+If you need to you can allow some users via [server authorization](/docs/server.md#authorize-to-impersonate) to impersonate the other users accounts, e.g.: Only admin users can impersonate to other users.
```
Comment (pradel) [inline website/docs/client.md:71]: Looks like the links are invalid and the website build is failing 🔴

**Review by pradel — CHANGES_REQUESTED** [review]
2 small comments and good to go!

File: website/docs/client.md (hunk 2 above)
Comment (BarbecueSilver) [reply website/docs/client.md]: Hope this is fixed now.

File: website/docs/server.md (hunk 1 above)
Comment (BarbecueSilver) [reply website/docs/server.md:147]: Moved it.

**Comment by BarbecueSilver** [comment]
@pradel all done :rocket: :partying_face:

**Comment by pradel** [comment]
@BarbecueSilver amazing, thanks 🚀

**Pull request merged**